src/hooks/tests/usePatchStatusSelect.test.ts

```diff
@@ -1,5 +1,5 @@
 import { renderHook, act } from "@testing-library/react-hooks";
-import { usePatchStatusSelect } from "hooks";
+import { useVersionTaskStatusSelect } from "hooks";
 import { waitFor } from "test_utils/test-utils";
```

Comment:
Can we replace the usage of `waitFor` with `waitForNextUpdate` Its the recommended way of waiting for react hook updates.